Assets/StreamingAssets/Quests/L0B00Y02.txt

```diff
@@ -79,8 +79,8 @@ QuestorPostfailure:  [1009]
 
 QuestLogEntry:  [1010]
 %qdt:
- _qgiver_ of the Dark Brotherhood
- in __qgiver_, ___qgiver_
+ _qgiver_ of the Dark
+ Brotherhood in ___qgiver_
  has me paying a visit on =assassin_,
```

Comment:
Minor question: From what I know of the quest scripting, it looks like the original text mentions the city and region where you got the quest from here, right? Was replaced with just one or the other? I don't see the reason for the change.

Reply:
I'm not sure, but Jay might have just made that change for everything to fit on a single line. The full location+region text might have been too many characters wide in some cases.